### PR TITLE
About/contact form

### PR DESCRIPTION
**Pull request by Kaiserabbas**
Add contact me and about me form

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
```
Comment (shahzaman900) [inline README.md:32]: Please add Live demo Option

**Review by shahzaman900 — COMMENTED** [review]
Please add live demo option

File: index.html [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+        </h1>
+        <p class="main-paragraph">
+          I'm a software developer! I can help you build a product, a feature or
+          a website. Look through some of my work and experience. If you like
+          what you see and have a project you need coded, don't hestiate to
+          contact me.
+        </p>
+    </section>
+    <section class="works">
+        <div class="card" style="width: 18rem;">
```
Comment (AjroudiRida) [inline index.html]: Avoid using inline styles in the project, instead use an external CSS file to integrate your styles.

**Review by AjroudiRida — COMMENTED** [review]
Highlights

Good PR
Good Github flows
Follow the best practice of HTML and CSS
Use semantic tags

**Required**

Avoid using inline styles in your Html document and use an external CSS file

**Pull request merged**